components/connector-service/internal/externalapi/managementinfohandler_test.go

```diff
@@ -146,4 +155,103 @@ func TestManagementInfoHandler_GetManagementInfo(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, http.StatusInternalServerError, rr.Code)
 	})
+
+	t.Run("should successfully get certificate info for application", func(t *testing.T) {
```

Comment:
The new test exercises exactly the same scenario as  "should successfully get management info urls for application", the only difference is in the checks done.I would suggest to enhance the existing test rather than adding a new one.

Reply:
Done!